ops/testing.py

```diff
@@ -531,6 +531,16 @@ def set_model_name(self, name: str) -> None:
             raise RuntimeError('cannot set the Model name after begin()')
         self._backend.model_name = name
 
+    def set_model_uuid(self, uuid: str) -> None:
```

Comment:
Is it better to add this to set_model_name or maybe calling that `set_model_info(name=None,uuid=None)`?

At the very least, shouldn't there be a test case that harness.set_model_uuid() causes harness.model.uuid to be updated? Would it be worth defaulting to a random UUID? (uuid4.uuid()) ?

Reply:
`set_model_name` already exists, so I did not want to break that API. I think it could make sense to add `set_model_info` as an overload.

Reply:
Even `set_model_name(self, name: str, model_uuid:str = None)` wouldn't break it, and it's simply a testing library

Reply:
The new commit offers an overload that adds the `set_model_info`, WDYT?